pkg/types/toml_config.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+type TomlConfig struct {
```

Comment:
One quick thing first, in Go you actually make abbreviations entirely uppercase (I've still not gotten used to this yet), so, if you were to leave this as this it'd actually be `TOMLConfig` BUT WAIT!

Does it matter that it's TOML? This is just the configuration for Tid, we read the TOML into it, but we could also use JSON too in the future if we really wanted. This struct on it's own has nothing to do with TOML, so probably shouldn't reference it at all (i.e. `types.Config`).